CDCgov/cdc-open-viz
# Feature/dev 8722

## Testing Steps
Testings steps for "highlight suppressed data in Gray" 
Open the "Bar Chart" ==> "Data Series"==> Click the "Add Special Bar" button and configure the settings to supress values.
Verify that the checkbox for "Highlight Suppressed Data in Gray" is enabled by default.
Check that the legend, tooltips, data table, and bar icons appear in gray until the checkbox is unchecked.
Testings steps for "Definition Link Below Legend" :
1. While you have added suppression you should by default see the new link " This Chart contsins suppressed data" hover or click on it and you will see additional information.
2. Note the link comes visible only if there is suppresseion added. Here is couple images to [show.](
<img width="1475" alt="Screenshot 2024-08-08 at 21 48 56" src="https://github.com/user-attachments/assets/1ca3b0c0-a88f-4a9d-a656-70785cd8a600">
<img width="1475" alt="Screenshot 2024-08-08 at 21 46 45" src="https://github.com/user-attachments/assets/4835b9ba-2b69-40b0-b950-3f4073be16c6">)



## Self Review

- I have added testing steps for reviewers
- I have commented my code, particularly in hard-to-understand areas
- My changes generate no new warnings
- New and existing unit tests are passing

## Screenshots (if applicable)



## Additional Notes
